codebuild/cd/deploy-snapshot.yml

```diff
@@ -22,12 +22,16 @@ phases:
     commands:
       - cd $CODEBUILD_SRC_DIR/aws-crt-java
       - export PKG_VERSION=$(git describe --tags | cut -f2 -dv)
+      - echo $PKG_VERSION
       - mkdir -p target/cmake-build/lib
       - aws s3 cp --recursive s3://aws-crt-java-pipeline/v${PKG_VERSION}/lib $CODEBUILD_SRC_DIR/aws-crt-java/target/cmake-build/lib
```

Comment:
trivial: can we re-order these so that we do all the stuff to put the libs in the right place, then do all the stuff to put the jars in the right place?

right now, the steps are kind of mixed together